apps/escrow/src/pages/home/create-wallet/CreateWallet.tsx

```diff
@@ -0,0 +1,42 @@
+import { Input, Button } from '@gear-js/ui';
+import { useForm } from '@mantine/form';
+import { isHex } from '@polkadot/util';
+import check from 'assets/images/icons/check.svg';
+import { CreateFormValues } from 'types';
+import styles from './CreateWallet.module.scss';
+
+type Props = {
+  onSubmit: (values: CreateFormValues) => void;
+};
+
+const initialValues = { buyer: '', seller: '', amount: '' };
+const validate = {
```

Comment:
Move to a constant, helpers

Reply:
imo it's more straightforward to keep it together with component